mmv1/products/looker/Instance.yaml

```diff
@@ -0,0 +1,362 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Instance'
+base_url: projects/{{project}}/locations/{{region}}/instances
+create_url: projects/{{project}}/locations/{{region}}/instances?instanceId={{name}}
+update_verb: :PATCH
```

Comment:
We'll want to use an update test to validate this works as expected: https://googlecloudplatform.github.io/magic-modules/how-to/add-handwritten-test/#update-tests

Reply:
Update test added (it uncovered a need for specific `update_mask_fields`).